src/keyframe-effect-constructor.js

```diff
@@ -144,9 +144,11 @@
     configurable: true,
     enumerable: true,
     value: function() {
+      window.document.timeline._updateAnimationsPromises();
```

Comment:
For another change, but we shouldn't be using 'window.ourdatastructure' inside the polyfill. We should have internal references to these structures, so that things still work if they're replaced by the author or some other polyfill.